UOP1_Project/Assets/Scripts/Systems/Settings/SettingsSystemAudioComponent.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class SettingsSystemAudioComponent : SettingsSystemSubComponents
+{
+    [SerializeField] Slider musicVolumeSlider;
+    [SerializeField] Slider sfxVolumeSlider;
+    
+    public float MusicVolume { get; private set; }
+    public float SfxVolume { get; private set; }
+
+    void Start()
+    {
+        Setup();
+    }
+
+    protected override void Setup()
+    {
+        //TODO: load previous music volume setting
+        MusicVolume = 50f;
+        musicVolumeSlider.SetValueWithoutNotify(MusicVolume);
+        //TODO: load previous sfx volume setting
+        SfxVolume = 50f;
+        sfxVolumeSlider.SetValueWithoutNotify(SfxVolume);
+    }
+    
+    #region UI CALLBACKS
+    
+    //TODO: clamp volume to [0, 1] or [0, 100]? Change Slider min max value in editor depending on use case
```

Comment:
Comments in code is a bad idea. IMHO it's better to create a ticket or a new pull request.

Reply:
[Open Projects - Conventions](https://docs.google.com/document/d/1-eUWZ0lWREFu5iH-ggofwnixDDQqalOoT4Yc0NpWR3k/edit)

Comments are fine as long as they're not redundent. And TODOs in particular are allowed as long as the functionality is not broken.

The real issue is the use of `#region`.

Reply:
Comments make code unreadable. No one will support comments actuality

Reply:
I think comments don't make the code unreadable. You were able to read and understand it obviously. Everyone uses comments and so do I. I even think they can contribute to make code more understandable in certain circumstances.

Also those conventions linked above were made after I created the PR and I didn't know that using regions, which unity itself and a lot of other companies make use of, is forbidden.

If so I will remove them, doesn't harm in any way either way.

Reply:
Just read Clean code by uncle Bob please

Reply:
I see. Your only issue is that I don't follow some code conventions of an individual (uncle bob), of which you think is "the only way". Thank you for your constructive comment, completely ignoring what I said. 👍